### PR TITLE
mdms host update

### PR DESCRIPTION
**Pull request by rahu-eGov**

### CODE REVIEW

File: deploy-as-code/helm/environments/pspcl-ifix-uat.yaml [hunk 1]
```diff
@@ -59,7 +59,7 @@ cluster-configs:
         fiscal-event-service: "http://fiscal-event-service.ifix:8080/"
         ifix-zuul: "http://ifix-zuul.ifix:8080/"
         ifix-department-entity-service: "http://ifix-department-entity-service.ifix:8080/"
-        egov-mdms-service: "http://egov-mdms-service.mgramseva:8080/"
+        egov-mdms-service: "https://mgramseva-uat.psegs.in/"
```
Comment (rushang7) [inline deploy-as-code/helm/environments/pspcl-ifix-uat.yaml:63]: Service host names should be read from config maps. 
Add the correct mdms host name in the config map, and change the helm chart to read the value from it.

**Review by rushang7 — CHANGES_REQUESTED** [review]

File: deploy-as-code/helm/environments/pspcl-ifix-uat.yaml (hunk 1 above)
Comment (rahu-eGov) [reply deploy-as-code/helm/environments/pspcl-ifix-uat.yaml:63]: added in egov-config mdms host

File: deploy-as-code/helm/environments/pspcl-ifix-uat.yaml [hunk 2]
```diff
@@ -52,14 +52,15 @@ cluster-configs:
         tracer-errors-provideexceptionindetails: "true"
         timezone: "Asia/Kolkata"
         druid-host: "druid-broker.olap"
+        egov-mdms-host: "https://mgramseva-uat.psegs.in/"
```
Comment (rushang7) [inline deploy-as-code/helm/environments/pspcl-ifix-uat.yaml:55]: Remove it from here. Service host names are part of egov-service-host config map.

File: deploy-as-code/helm/environments/pspcl-ifix-uat.yaml (hunk 1 above)
Comment (rushang7) [reply deploy-as-code/helm/environments/pspcl-ifix-uat.yaml:63]: Why isn't "http://egov-mdms-service.mgramseva:8080/" this working?

**Pull request closed without merging**